src/DependencyInjection/Compiler/AdminSearchCompilerPass.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\DependencyInjection\Compiler;
+
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Exception\LogicException;
+
+/**
+ * This class configures which admins must be considered for global search at `SearchHandler`.
+ *
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ */
+final class AdminSearchCompilerPass implements CompilerPassInterface
+{
+    public const TAG_ATTRIBUTE_TOGGLE_SEARCH = 'global_search';
+
+    private const TAG_DMIN = 'sonata.admin';
```

Comment:
```suggestion
    private const TAG_ADMIN = 'sonata.admin';
```

Reply:
And this tag is used in multiple compiler pass. It should be the same constant, isn't it ?

Reply:
Yes, I see this tag is used in other classes, but since we haven't defined a global place where it could be defined, I decided to use this only for the purposes of this class.

Reply:
Should we declare this in the AdminInterface ?

Reply:
IMO, we should not tie the classes outside the Symfony specific DI scope with something related to that configuration.

Reply:
Then I think the tag should be in `AddDependencyCallsCompilerPass` which should mention Admin in the name (and maybe be splitted ?).

Reply:
I'll add the constant at `AddDependencyCallsCompilerPass` in a new pedantic PR.

Reply:
I have added it here: https://github.com/sonata-project/SonataAdminBundle/pull/6614

WDYT ?